docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -509,7 +514,14 @@ cause the task to fail, set the field `pre_error_fails_task` to true.
 Likewise, setting the field `post_error_fails_task` to true will enforce
 that failures in `post` cause the task to fail.
 
-``` yaml
+If pre_error_fails_task is set to true and pre fails, it will exit early 
+and not run the main task commands, but it will still run the post block.
+
+If post_error_fails_task is set to true and both the main task and post 
+block fail, the failing command in the main task will be displayed as the 
+failing command, not the failing post command. 
```

Comment:
Can we also mention that `pre_error_fails_task` and `post_error_fails_task` has no effect for tasks in task groups? Users can instead use the analogous task group YAML settings.

Reply:
done